examples/blocks/.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = tab
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[package.json]
+indent_style = space
+indent_size = 2
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Please remove this file. We have our own code styling rules and those are in the root directory :) Same thing for `.gitattributes` :)